ui/packages/consul-ui/tests/acceptance/dc/kvs/create.feature

```diff
@@ -49,3 +49,27 @@ Feature: dc / kvs / create
     And I click create
     And I see the text "New Key / Value" in "h1"
     And I see the text "key-value" in "[data-test-breadcrumbs] li:nth-child(2) a"
+    And I see the "[data-test-kv-key]" element
+  Scenario: Clicking create from within a just created folder
+    Given 1 datacenter model with the value "datacenter"
+    When I visit the kv page for yaml
+    ---
+      dc: datacenter
+    ---
+    Then the url should be /datacenter/kv/create
+    And the title should be "New Key / Value - Consul"
+    Then I fill in with yaml
+    ---
+      additional: key-value/
+    ---
+    Given 1 kv model from yaml
+    ---
+    - key-value/
+    ---
+    And I submit
+    Then the url should be /datacenter/kv
+    And I click "[data-test-kv]"
+    And I click "[data-test-create]"
+    And I see the text "New Key / Value" in "h1"
+    And I see the text "key-value" in "[data-test-breadcrumbs] li:nth-child(2) a"
+    And I see the "[data-test-kv-key]" element
```

Comment:
Test was mainly copy/pasted from the above (not visible unless you view the entire file). We were already testing: "creating a folder" and "clicking create from within a folder" but not "creating a folder and then immediately clicking create form within that folder".

I also noticed that we weren't asserting for existence of the input field in the "clicking create from within a folder" test that already existed. Whilst this test wasn't failing even with the extra assertion, I thought it would be a good idea to add here to helper prevent any possible regression in the future (line 52)